charts/connect/docs/fork-pr-testing.md

```diff
@@ -0,0 +1,13 @@
+# Fork PR Testing Guide
```

Comment:
I don't see any fork workflows. Will this be added in a future PR? If so, then it makes more sense to add this as part of that PR.